setup.cfg

```diff
@@ -18,3 +18,11 @@ based_on_style = google
 
 [metadata]
 license_files = LICENSE
+
+[pytype]
+inputs =
+    src/python/grpcio/grpc/experimental
+    src/python/grpcio_tests/tests_aio
+
+disable =
+    import-error
```

Comment:
What's the reasoning behind this?

Reply:
Sadly, the `pytype` doesn't understand `grpc._cython.cygrpc`.